src/main/scala/com/coralogix/sbtprotodep/backends/ProtodepBinary.scala

```diff
@@ -26,12 +26,21 @@ class ProtodepBinary(
     }
   }
 
-  def fetchProtoFiles(root: File, forced: Boolean, cleanup: Boolean, https: Boolean): Unit = {
+  def fetchProtoFiles(root: File, locked: Boolean, https: Boolean): Unit = {
     val args =
       List(
-        if (forced) Some("-f") else None,
-        if (cleanup) Some("-c") else None,
-        if (https) Some("-u") else None
+        Some("--cleanup"),
```

Comment:
if this `locked` flag is primarily meant for having different behaviours for local dev and for CI i wonder about two things
1. why not to call it `ci` instead of `locked`
2. do i wanna run cleanup every time when working locally?
WDYT?

Reply:
yeah in protofetch it's called `--locked` but protodep doesn't have a exact counterpart i think, maybe default `protodep up` (without `-f`) would be somehow similar...

that's kind of my point, base trait was changed, which is imo definitive improvement, but we got rid of properties which doesn't fit protofetch (forced, cleanup) and added property which is incompatible with protodep (locked), hence the wondering, can we come up with the signature fitting both based on usage perhaps?

Reply:
Ah, you mean renaming the trait parameter to `ci`? Yes, this makes sense, will do.